test/unit/spec/EnrollSms_spec.js

```diff
@@ -608,6 +618,30 @@ function (Okta, OktaAuth, LoginUtil, Util, AuthContainer, Form, Beacon, Expect,
             ]);
           });
       });
+    }
+
+    Expect.describe('Header & Footer', function () {
+      testHeaderAndFooter(resAllFactors, setupAndSendValidCode, 'testStateToken');
+    });
+
+    Expect.describe('Header & Footer with idx pipeline', function () {
```

Comment:
for all the tests here, the name (just or consistency sake): `.. on Idx Pipeline`